src/hooks/ga.hooks.ts

```diff
@@ -0,0 +1,19 @@
+import { useEffect } from 'react';
+import { useLocation } from 'react-router-dom';
+
+const GA_TRACKING_ID = 'G-YSLD6T408C';
+
+function usePageViews() {
+  const location = useLocation();
+
+  useEffect(() => {
+    (window as any).gtag('config', GA_TRACKING_ID, {
+      page_path: location.pathname + location.search,
```

Comment:
Does it record the URL query? Maybe later, we'll make the share feature using query.

Reply:
Okay. When site entered it only logs entry paths, not other details like URL queries.